test/logger.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env mocha */
+'use strict';
+const Winston = require('winston');
+
+describe('logging', () => {
+  it('returns a WINSTON object', () => {
+    const config = require('../lib/config');
+    const log = require('../lib/logger').attach(config);
+
+    log.should.be.an.instanceOf(Winston.Logger);
```

Comment:
Instead of testing the logger type, we can test logger behavior.

Pass in a `Config` compatible object that logs to a temp file and check the following:
* That the logger defines convenience methods for `info`, `warn`, and `error` logging
* That logged messages show up in the temp file
* If the log level is set to `error`, messages logged at `info` and `warn` level don't show up